R/Design-methods.R

```diff
@@ -209,7 +209,7 @@ setMethod("simulate",
              truth, args = NULL, firstSeparate = FALSE,
              mcmcOptions = McmcOptions(),
              parallel = FALSE, nCores =
-               min(parallel::detectCores(), 5),
+               min(parallel::detectCores(), 5), derive = NULL,
```

Comment:
I think the default could be an epmty list.

```suggestion
               min(parallel::detectCores(), 5), derive = list(),
```